File: Code-Chef/Chef-and-Operators/chef_and_operators.c
/*  Problem Statement: https://www.codechef.com/problems/CHOPRT
    Author: striker
*/

#include<stdio.h>
#include<stdlib.h>
#include<assert.h>

int main(void) {
    size_t test;
    scanf("%lu", &test);
    assert(test > 0 && test < 10001);
    while(test--) {
        size_t a,b;
        scanf("%lu%lu", &a,&b);
        assert((a > 0 && a <= 1000000001) && (b > 0 && b <= 1000000001));
        if(a == b) {
            printf("=\n");
        } else if(a > b) {
            printf(">\n");
        } else {
            printf("<\n");
        }
    }
    return 0;
}

File: Code-Chef/Chef-and-Operators/chef_and_operators.py
'''
Problem Statement: https://www.codechef.com/problems/CHOPRT
Author: striker
'''

def main():
    test = int(input())
    for _ in range(test):
        a,b = tuple(map(int,input().split()))
        if a == b:
            print("=")
        elif a > b:
            print(">")
        else:
            print("<")

if __name__ == "__main__":
    main()